pai-management/bootstrap/hadoop-service/hadoop-configuration/nodemanager-generate-script.sh

```diff
@@ -34,3 +34,11 @@ sed  -i "s/{HDFS_ADDRESS}/${HDFS_ADDRESS}/g" $HADOOP_CONF_DIR/core-site.xml
 
 sed  -i "s/{LOGSERVER_ADDRESS}/${LOGSERVER_ADDRESS}/g" $HADOOP_CONF_DIR/mapred-site.xml 
 
+# set memory and cpu resource for nodemanager
+mem_available=`cat /proc/meminfo | grep "MemAvailable" | awk '{print $2}'`
```

Comment:
`MemAvailable` changes from time to time. Maybe `MemTotal` is a better choice?

Reply:
OK, a fix value for this is better than a dynamic value.